access-mgmt-service-registry/README.adoc

```diff
@@ -0,0 +1,143 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:org-name: Application Services
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: OpenShift Streams for Apache Kafka
+:product: Streams for Apache Kafka
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:getting-started-service-registry-url: getting-started-service-registry/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:quarkus-service-registry-url: quarkus-service-registry/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-kafka-url: rhoas-cli-kafka/README.adoc
+:rhoas-cli-service-registry-url: rhoas-cli-service-registry/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+:access-mgmt-url: access-mgmt/README.adoc
+:access-mgmt-service-registry-url: access-mgmt-service-registry/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-service-registry-access"]
+= Managing account access in {registry-product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access-service-registry
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a {registry} instance, administrator, or organization administrator in {registry-product-long}, you can manage the level of access that other user accounts and service accounts have to your {registry} instance. You can grant or remove access to your instance for specific user accounts in your organization based on user roles. You can also allow other users or service accounts to manage the level of access to your instance for you.
+
+{registry} instance owners or administrators can manage access for only the {registry} instances that they create or for instances that the owner has allowed them to access and change. Organization administrators can manage access for all {registry} instances.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
```

Comment:
FYI - This problem is going to be solved soon (it was released in underlying library)